styles.css

```diff
@@ -4,7 +4,6 @@
   margin: 0;
   padding: 0;
   scroll-behavior: smooth;
-  /* outline: 1px solid red; */
 }
 
```

Comment:
Kindly add the same style provided in your Figma template (Typography):
![Screen Shot 2021-08-31 at 4 54 21](https://user-images.githubusercontent.com/22258274/131482705-b2c201a7-36c3-4f5c-88de-c08489f8b58a.png)

As well improve your project giving the same font 
![Screen Shot 2021-08-31 at 4 55 09](https://user-images.githubusercontent.com/22258274/131483043-4ae5a047-0685-40f2-ac25-d36d8201ba0f.png)
color than you style requested:

Reply:
Actually, due to the accessibility factors, I have changed the colour to black. hope it's fine.

Reply:
That accessibility it's mentioned on your project requirements?

Reply:
No not on this project, but I did that in last week project, so I thought I have to keep that in mind.

Reply:
Don't worry!... Keep coding hard bro! :D